Modules/_decimal/_decimal.c

```diff
@@ -4330,16 +4330,7 @@ dec_##MPDFUNC(PyObject *self, PyObject *Py_UNUSED(dummy))   \
 
 /* Boolean function with an optional context arg. */
```

Comment:
To improve clarity over what arguments are expected in the function definition:
```suggestion
/* Boolean function with an optional context arg. 
   Argument Clinic provides PyObject *self, PyObject *context
*/
```

Reply:
Nice idea.